CopilotKit/packages/react-textarea/src/types/base/base-autosuggestions-config.tsx

```diff
@@ -76,10 +76,20 @@ const defaultShouldToggleHoveringEditorOnKeyPress = (
   event: React.KeyboardEvent<HTMLDivElement>,
   shortcut: string,
 ) => {
-  // if command-k, toggle the hovering editor
-  if (event.key === shortcut && event.metaKey) {
-    return true;
+  const isWindowsOs = navigator.userAgent.includes("Windows");
```

Comment:
Recommend refactoring the OS-specific shortcut logic to be more concise:

```javascript
// Check if it's the keyboard combo we want
const isMac = /Mac|iPod|iPhone|iPad/.test(navigator.platform);
const modifierKey = isMac ? event.metaKey : event.ctrlKey;

return event.key.toLowerCase() === shortcut.toLowerCase() && modifierKey;
```

Rationale:
- Eliminates duplicate code paths for Windows/Mac
- More maintainable with a single return statement
- Case-insensitive comparison prevents bugs
- Uses ternary operator for clean modifier key selection
- Still captures the same functionality in fewer lines